SwitchItem/SwitchItem.module.less

```diff
@@ -9,7 +9,7 @@
 	}
 
 	.switchIcon {
-		margin-left: 10px;
+		.margin-start-end(9px, 0);
```

Comment:
Hello @daniel-stoian-lgp 
Is there any reason why there is 9px instead of 10px?

Reply:
Yes, it's because of the resolution independence functionality. The rule is to use multiple of 3 values, for values above 3px

Reply:
Then it is reasonable to me.